internal/cmd/commands/scopescmd/list_key_version_destruction_jobs.go

```diff
@@ -0,0 +1,156 @@
+package scopescmd
+
+import (
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/boundary/api"
+	"github.com/hashicorp/boundary/api/scopes"
+	"github.com/hashicorp/boundary/internal/cmd/base"
+	"github.com/mitchellh/cli"
+	"github.com/mitchellh/go-wordwrap"
+	"github.com/posener/complete"
+)
+
+var (
+	_ cli.Command             = (*ListKeyVersionDestructionJobsCommand)(nil)
+	_ cli.CommandAutocomplete = (*ListKeyVersionDestructionJobsCommand)(nil)
+)
+
+type ListKeyVersionDestructionJobsCommand struct {
+	*base.Command
+}
+
+func (c *ListKeyVersionDestructionJobsCommand) Synopsis() string {
+	return wordwrap.WrapString("List all pending key version destruction jobs within a scope", base.TermWidth)
+}
+
+func (c *ListKeyVersionDestructionJobsCommand) Help() string {
+	return base.WrapForHelpText([]string{
+		"Usage: boundary scopes list-key-version-destruction-jobs [args]",
+		"",
+		"  List all pending key version destruction jobs within a scope. A key version",
+		"  destruction job asynchronously re-encrypts any existing data with the latest",
+		"  key version before destroying the key version. Example:",
+		"",
+		`    $ boundary scopes list-key-version-destruction-jobs -scope-id global`,
+		"",
+		"",
+	}) + c.Flags().Help()
+}
+
+func (c *ListKeyVersionDestructionJobsCommand) Flags() *base.FlagSets {
+	set := c.FlagSet(base.FlagSetHTTP | base.FlagSetClient | base.FlagSetOutputFormat)
+	f := set.NewFlagSet("Command Options")
+
+	f.StringVar(&base.StringVar{
+		Name:   "scope-id",
+		Target: &c.FlagScopeId,
+		Usage:  "The id of the scope in which to list key version destruction jobs",
+	})
+
+	return set
+}
+
+func (c *ListKeyVersionDestructionJobsCommand) AutocompleteArgs() complete.Predictor {
+	return complete.PredictAnything
+}
+
+func (c *ListKeyVersionDestructionJobsCommand) AutocompleteFlags() complete.Flags {
+	return c.Flags().Completions()
+}
+
+func (c *ListKeyVersionDestructionJobsCommand) printListTable(items []*scopes.KeyVersionDestructionJob) string {
+	if len(items) == 0 {
+		return "No keys version destruction jobs found"
+	}
+	output := []string{
+		"",
+		"Key version destruction job information:",
+	}
+	for i, item := range items {
+		if i > 0 {
+			output = append(output, "")
+		}
+		if true {
+			output = append(output,
+				fmt.Sprintf("  Key version ID:    %s", item.KeyVersionId),
+				fmt.Sprintf("    Scope ID:        %s", item.Scope.Id),
+			)
+		}
+		if !item.CreatedTime.IsZero() {
+			output = append(output,
+				fmt.Sprintf("    Started:         %s", item.CreatedTime.Local().Format(time.RFC1123)),
+			)
+		}
+		if item.Status != "" {
+			output = append(output,
+				fmt.Sprintf("    Status:          %s", item.Status),
+			)
+		}
+		if true {
+			output = append(output,
+				fmt.Sprintf("    Rows completed:  %d", item.CompletedCount),
+			)
+		}
+		if true {
+			output = append(output,
+				fmt.Sprintf("    Total rows:      %d", item.TotalCount),
+			)
+		}
+	}
```

Comment:
This is a proposal for how to display these on the CLI. This is what it looks like on the CLI:

```
$ boundary scopes list-key-version-destruction-jobs -scope-id global

Key version destruction job information:
  Key version ID:    kdkv_OJz1sJIgbZ
    Scope ID:        global
    Started:         Mon, 26 Sep 2022 17:23:07 PDT
    Status:          pending
    Rows completed:  0
    Total rows:      300
```